examples/simple/Error_Codes.csv

```diff
@@ -95,6 +95,14 @@ UB-STDLIB1,"Called realloc on memory not allocated by malloc, or already freed."
 UB-STRING1,Null pointer passed to strcpy.,"7.1.4:1, J.2:1 item 108",Undefined Behavior
 UB-STDLIB2,"Called free on memory not allocated by malloc, or already freed.","7.22.3.3:2, J.2:1 item 179",Undefined Behavior
 UB-TTE1,program requires the formation of a composite type from a variable length array type whose size is specified by an expression that is not evaluated.,"6.2.7:3, J.2:1 item 16",Undefined Behavior
+UB-PTHREAD1,Initializing an already initialiized mutex,POSIX 2008 sec. pthread_mutex_init,Undefined Behavior
```

Comment:
I'd like to see an extra column with like an X or something in it for those errors that are only caught with RV-Match.